src/NextGen/DonationForm/Blocks/DonationFormBlock/resources/app/utilities/registerFieldAndBuildProps.tsx

```diff
@@ -2,23 +2,26 @@ import {Field} from '@givewp/forms/types';
 import {FieldProps} from '@givewp/forms/propTypes';
 import {UseFormReturn} from 'react-hook-form';
 import buildRegisterValidationOptions from './buildRegisterValidationOptions';
-import {getFieldErrorTemplate, getFieldLabelTemplate} from '../templates';
 import getErrorByFieldName from './getErrorByFieldName';
+import {withTemplateWrapper} from '@givewp/blocks/form/app/templates';
+import getFormDesign from '@givewp/blocks/form/app/utilities/getFormDesign';
+
+const formDesign = getFormDesign();
+const LabelTemplate = withTemplateWrapper(formDesign.layouts.fieldLabel);
+const ErrorMessageTemplate = withTemplateWrapper(formDesign.layouts.fieldError);
```

Comment:
Wrapping these feels like markup overkill.

Reply:
Both of these are currently using the node wrapper from the previous `getTemplate()` function.  I agree it _may_ be overkill but there's also something to say about consistency of the template wrapper being used that we should  discuss.

Reply:
Mm... For these very specific layouts it just feels like markup cruft. I appreciate the consistency when it comes to fields, elements, and such, but I don't want to follow the convention for its own sake. I'm more interested in what the resulting markup is, and in this case it's too much.